content/docs/studio/troubleshooting.md

```diff
@@ -38,10 +38,14 @@ files that contain the metrics and hyperparameters that you want to visualize.
 Otherwise, you will get this message when you try to add a project:
 `We could not find data to visualize in this repo`.
 
-For more details, refer to the section on
-[how to prepare your Git repositories for use with Iterative Studio](/doc/studio/user-guide/prepare-repositories).
-Instructions on how to specify custom files can be found
-[here](/doc/studio/user-guide/projects/project-settings#configuring-project-settings).
+For more details, refer to the section on how to [prepare your Git repositories]
+for use with Iterative Studio. Instructions on how to specify custom files can
+be found [here].
+
+[prepare your git repositories]:
+  /doc/studio/user-guide/prepare-your-repositories
+[here]:
```

Comment:
I wouldn't suggest making a separate entry for "here" this way. Coz "here" isn't really meaningful outside the immediate context.

Reply:
Renamed `[project-settings]`.